Prace_domowe/Praca_domowa6/Gryszkiewicz_Maciej/PD6_Maciej_Gryszkiewicz.ipynb

```diff
@@ -0,0 +1,816 @@
+{
```

Comment:
<p>dobrze, że zadbano o to, aby po przekształceniach wartości pikseli pozostały w przedziale [0,1]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>